contracts/transfer/circuits/src/dusk_contract/execute.rs

```diff
@@ -0,0 +1,1508 @@
+        // Derive the one time secret key, sk_r, for the note
+        let sk_r1 = ssk1.sk_r(note1.stealth_address());
+        // Derive the one time public key, pk_r, for the note
+        let pk_r1 = note1.stealth_address().pk_r();
+        let nullifier1 = note1.gen_nullifier(&ssk1);
+        let input_note_value_one = JubJubScalar::from(600 as u64);
+        let input_commitment_one = compute_value_commitment(
+            input_note_value_one,
+            input_note_blinder_one,
+        );
+
+        // Generate the (a,b) for the note
+        let secret3 = JubJubScalar::from(300 as u64);
+        let secret4 = JubJubScalar::from(400 as u64);
+        // Declare the secret spend key for the note
+        let ssk2 = SecretSpendKey::new(secret3, secret4);
+        // Derive the public spend key from the note from the above secret spend key
+        let psk2 = PublicSpendKey::from(ssk2);
+        // Assign the value of the first note as 400, which is incorrect
+        let value2 = 200u64;
+        let r2 = JubJubScalar::from(450 as u64);
+        let nonce2 = JubJubScalar::from(6750 as u64);
+        let input_note_blinder_two = JubJubScalar::from(200 as u64);
+        // Create a deterministic note so that we can assign the blinder and not have inner randomness
+        let mut note2 = Note::deterministic(
+            NoteType::Transparent,
+            &r2,
+            nonce2,
+            &psk2,
+            value2,
+            input_note_blinder_two,
+        );
+        // Set the position of the note
+        let pos2 = note2.set_pos(1);
+        // Define the hash of the note, which is required for the preimage check
+        let note_hash2 = note2.hash();
+        let pos_b = note2.pos();
+        // Derive the one time secret key, sk_r, for the note
+        let sk_r2 = ssk2.sk_r(note2.stealth_address());
+        // Derive the one time public key, pk_r, for the note
+        let pk_r2 = note2.stealth_address().pk_r();
+        // Assign the nullifier of the note
+        let nullifier2 = note2.gen_nullifier(&ssk2);
+        let input_note_value_two = JubJubScalar::from(200 as u64);
+        // Generate the value commitment of the note from the value and blinder
+        let input_commitment_two = compute_value_commitment(
+            input_note_value_two,
+            input_note_blinder_two,
+        );
+
+        let mut tree =
+            PoseidonTree::<Note, PoseidonAnnotation, Blake2b>::new(17);
+        // Assign the postitions of the notes to a position in the tree
+        let tree_pos_1 = tree.push(note1)?;
+        let tree_pos_2 = tree.push(note2)?;
+
+        // Generate the crossover commitment, C.c(v,b)
+        let crossover_commitment_value = JubJubScalar::from(200 as u64);
+        let crossover_commitment_blinder = JubJubScalar::from(100 as u64);
+        let crossover_commitment = compute_value_commitment(
+            crossover_commitment_value,
+            crossover_commitment_blinder,
+        );
```

Comment:
It happens to be those values but I picked them due to ease. They must satisfy `input_note_value - output_note_value - crossover_value - fee = 0` and its 800 - 400 - 200 - 200 but it could also be 900 - 100 - 300 - 500. So they wouldn't be reusable.